Resources/config/services.yml

```diff
@@ -97,3 +99,19 @@ services:
         class: %ezdemo.installer.demo_installer.class%
         tags:
             - {name: ezplatform.installer, type: demo}
+
+    ezdemo.ez_content_banner_factory:
+        class: %ezdemo.privacy_cookie.banner_factory.class%
+        factory: ["%ezdemo.privacy_cookie.banner_factory.class%", build]
+        arguments:
+            - contentId: %ezdemo.privacy_cookie.content_banner.content_id%
```

Comment:
Maybe this could be declared as a container variable, and used here. Would be more readable. The array could be in default_settings.yml.